src/pages/auth.js

```diff
@@ -78,28 +125,37 @@ export class Auth extends React.Component {
 
     const registration_success = ui.registrationSuccess;
 
+    // also classes: parallax parallax__layer-pseudo parallax__layer-bg_pseudo
+
+    /*
+      <header ref="landingClone" className="landing__body landing__body-shortened hidden">
+        <p className="layout__row layout__row-small landing__small_title" style={{ position: 'relative', left: 4 }}>Welcome to LibertySoil.org</p>
+        <h1 ref="subtitle" className="landing__subtitle landing__subtitle-narrow">Education change network</h1>
+      </header>
+    */
+
     return (
-      <div className="page__container-bg font-open_sans font-light">
-        <section className="landing landing-big landing-bg landing-bg_house">
+      <div className="page__container-bg font-open_sans font-light ">
+        <section ref="landing" className="landing landing-big landing-bg landing-bg_house landing-bg_fixed layout__row-group layout__row-full">
```

Comment:
> do we use "common" properies like this.something instead of this.state.something?

it depends. they serve different purposes. `this.state` change can be used to trigger re-rendering. "common" properties are not taken into account when rerenderability is calculated